eureka-core/src/main/java/com/netflix/eureka/util/EurekaMonitors.java

```diff
@@ -87,11 +86,11 @@ private EurekaMonitors(String name, String description) {
         }
     }
 
-    @com.netflix.servo.annotations.Monitor(name = "count", type = DataSourceType.COUNTER)
```

Comment:
This seems like a curiously named metric "count", is there something that gets added with the servo Monitor that allows us to disambiguate such a generically named id @brharrington?

Reply:
It would have a dimenion `class=EurekaMonitors`. See: https://github.com/Netflix/servo/blob/master/servo-core/src/main/java/com/netflix/servo/monitor/Monitors.java#L264

Reply:
Got it, should I add this dimension to the metrics?  Will there be any gotchas about emitting that dimension manually?

Reply:
For better compatibility with the previous data it would be good to add it explicitly. There shouldn't be any issues with adding it directly.

Reply:
Will go ahead and do that to attempt to avoid any external changes in the metrics.  Thanks @brharrington